### PR TITLE
SEOs, Typo Alerts

### PR DESCRIPTION
**Pull request by Shruti3004**
- Meta tags are added to all the index.html for SEO, this will help other websites to know about what exactly this site is for. 
- Fixed the typos.

### CODE REVIEW

**Comment by rahul2240** [comment]
1. Better use commit message as 
"add something to something", "fix something in something" etc.

2. Add commit description 
"it should tell what commit is doing or maybe why you did this"

3. 2nd commit has nothing, try removing the extra commit or you can ask for help.

**Comment by Shruti3004** [comment]
Okay sir
I will do the same.

**Comment by rajat2502** [comment]
Also, make different PRs for different issues/tasks.

**Comment by Shruti3004** [comment]
okay sir

File: index.html [hunk 1]
```diff
@@ -11,6 +11,34 @@
     </script>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+    <meta name="description" content="Nibble Computer Society">
```
Comment (rajat2502) [inline index.html]: Repeated below

File: index.html [hunk 2]
```diff
@@ -11,6 +11,34 @@
...
+    <meta name="keywords" content="Nibble Computer Society is the official computer society of JSS Noida. We are a group of students working towards enhancing knowledge in the field of computer science. The website shows our contribution to the college, our members and events & workshops that we organize."/>
```
Comment (rajat2502) [inline index.html]: These are not keywords. Keywords can be Nibble Computer Society, NCS, JSS Noida, JSS, student society, computer society, etc

File: index.html [hunk 3]
```diff
@@ -11,6 +11,34 @@
...
+    <meta name="HandheldFriendly" content="True"/>
+    <meta name="MobileOptimized" content="320"/>
```
Comment (rajat2502) [inline index.html]: remove this line.

**Review by rajat2502 — CHANGES_REQUESTED** [review]

**Comment by Shruti3004** [comment]
Okay sir

File: index.html (hunk 1 above)
Comment (rajat2502) [reply index.html]: Better remove this one, as it does not contain the description.

**Comment by Shruti3004** [comment]
Okay sir
I will make the mentioned changes.

**Comment by rajat2502** [comment]
Any updates?

**Comment by Shruti3004** [comment]
@rajat2502 Sir I have made the changes please review it.

**Comment by rajat2502** [comment]
Add these meta tags to all the other pages as well. The content of the tags should be changed accordingly.

File: team/index.html [hunk 4]
```diff
@@ -12,6 +12,31 @@
 	</script>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+	<meta name="keywords" content="Nibble Computer Society, NCS, JSS Noida, JSS, NCS JSS, Student society, Computer society"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="NCS is a student group that constantly tries to innovate and foster technical activities in the campus. Since 2000, we have focused on creating meaningful experiences through immersive learning. We hope to bring the best in code, development and design to the college scene."/>
```
Comment (rajat2502) [inline team/index.html:17]: @krrohitch @rex12543 Is this fine as the description of the team page?

File: project/index.html [hunk 5]
```diff
@@ -12,6 +12,31 @@
   </script>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+  <meta name="keywords" content="Nibble Computer Society, NCS, JSS Noida, JSS, NCS JSS, Student society, Computer society"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="We hope to bring the best in code, development and design to the college scene."/>
```
Comment (rajat2502) [inline project/index.html:17]: Also, this one.

**Pull request closed without merging**